nss-day-cohort-41/tabloid-fullstack-shocking-secret-keepers
# Sf last day

# Description
Regarding ticket #31 #32
1. Currently able to post a subscription to database, however logic for subscribe and unsubscribe buttons not working so will show both buttons for everyone's post details page including the user logged in
2. Currently after posting a subscription to the database, on the homepage should be able to see a list of posts
3. Unsubscribing to author is currently not working

## Type of change
Please delete options that are not relevant.
# Testing Instructions
1. fetch --all
2. run 01_Tabloid_Create_DB.sql to add new property to subscription table
3. Run 02_Tabloid_Seed_Data.sql 
4. run Tabloid server
5. npm start
6. log in as a user and initially you should not see anything in the homepage in regards to subscribed author posts because no user is subscribed yet
7. go to details page for a post that is not currently yours and click subscribe to author
8. go to Tabloid sql server and query for SELECT * FROM Subscription 
9. You should see your subscription 
10. go to home page of Tabloid app and you should see your subscribed posts there 
# Checklist:
